### ISSUE TITLE
[Bug]: [date 11.30]tke regression: sysbench 100w update test reported w-w conflict

### ISSUE THREAD

**Issue by heni02** [matrixorigin/matrixone#13150]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):f8dba6beb64044147e6415497f2107a3fe347603
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

job:https://github.com/matrixorigin/mo-nightly-regression/actions/runs/7047941218/job/19201992829
<img width="1413" alt="企业微信截图_433a7608-5d44-4173-8ba3-8c9bfb313f08" src="https://github.com/matrixorigin/matrixone/assets/113406637/385b7f16-953e-4305-b568-a3d94ededeb9">

mo log:
http://132.232.112.34:30088/explore?panes=%7B%22dVe%22:%7B%22datasource%22:%22loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bnamespace%3D%5C%22mo-nightly-regression-20231130%5C%22%7D%20%7C%3D%20%60%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki%22%7D,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%22now-12h%22,%22to%22:%22now%22%7D%7D%7D&schemaVersion=1&orgId=1

**Comment by YANGGMM** [comment]
@zhangxu19830126 , plz help to solve this issue

**Comment by zhangxu19830126** [comment]
还在查，很难复现，几乎复现不了

**Comment by zhangxu19830126** [comment]
maybe caused by #13136

**Comment by XuPeng-SH** [comment]
it is definitely has nothing to do with #13136. #13136 was merged at 12/05 and this issue was reported at 12/01. Please do some investigate before transfering the issue.

**Comment by XuPeng-SH** [comment]
#13136 was merged at 12/05 and this issue was reported at 12/01.

**Comment by zhangxu19830126** [comment]
还没有进展，但是jiangwei使用一个repo在复现。

**Comment by zhangxu19830126** [comment]
还没有进展，但是 jiangwei 使用一个 repo 在复现。

**Comment by zhangxu19830126** [comment]
maybe fixed by #13644

**Comment by heni02** [comment]
最近回归没有出现
commit：nightly-68e06c146
date 12.25回归job：https://github.com/matrixorigin/mo-nightly-regression/actions/runs/7322467586/job/19944061421
